src/website/webview/urls.py

```diff
@@ -1,8 +1,16 @@
-from django.urls import path
+from django.urls import path, re_path
 
-from webview.views import HomeView
+from webview.views import HomeView, NixpkgsIssueView, NixpkgsIssueListView
 
 app_name = "webview"
+
+
 urlpatterns = [
     path("", HomeView.as_view(), name="home"),
+    path("issues/", NixpkgsIssueListView.as_view(), name="issue_list"),
```

Comment:
Sure, but that's pretty much a standard https://security.archlinux.org/AVG-2596 FWIW.